src/pages/LandingPage.js

```diff
@@ -51,11 +52,11 @@ const makeRightArrowWithBackgroundIcon = () =>
     [icon('arrowRight', { color: 'white' })]
   );
 
-const makeCard = (link, title, body) =>
+const makeCard = _.map(({ link, title, body, linkPathParams, linkQueryParams }) =>
```

Comment:
It's confusing that the name of this function suggests it deals with a singular card, but it actually creates multiple cards.

```suggestion
const makeCards = _.map(({ link, title, body, linkPathParams, linkQueryParams }) =>
```